webdriverio/index.d.ts

```diff
@@ -352,36 +352,37 @@ declare namespace WebdriverIO {
         ): Client<P>;
         getHTML<P>(includeSelectorTag: boolean): Client<P>;
 
-        getLocation(selector?: string): Size;
-        getLocation(selector: string, axis: string): number;
-        getLocation(axis: string): number;
+        getLocation(selector?: string): Location;
+        getLocation(selector: string, axis: Axis): number;
+        getLocation(axis: Axis): number;
```

Comment:
This doesn't look right - `Axis` is just a union of two strings, but selector will already catch this. What's more, [I don't see this documented.](http://webdriver.io/api/property/getLocation.html)

You should probably get rid of this overload.